src/ejabberd_oauth.erl

```diff
@@ -521,7 +521,7 @@ process(_Handlers,
     Top =
         ?DIV(<<"section">>,
              [?DIV(<<"block">>,
-                   [?A(<<"https://www.ejabberd.im">>,
+                   [?A(<<"https://ejabberd.im/">>,
```

Comment:
No, because all those https://ejabberd.im/ links redirect to https://www.ejabberd.im/